modules/next/modules/next_extras/next_extras.info.yml

```diff
@@ -1,7 +1,7 @@
 name: Next.js Extras (Experimental)
 description: Adds extra (mostly experiemental) functionality to Next.js module
 type: module
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: ^8.8 || ^9 || ^10
```

Comment:
We're planning to drop support for Drupal 8 when we go 10.

Reply:
Hi @shadcn , Done with the changes 
Also rebased my forked repo with main repo and resolved the conflicts file.
It is ready for review and do the needful.